main/src/library/Nel.flix

```diff
@@ -57,6 +57,11 @@ instance Functor[Nel] {
     pub def map(f: a -> b & ef, xs: Nel[a]): Nel[b] & ef = Nel.map(f, xs)
 }
 
+// TODO implement Applicative
+// instance Monad[Nel] {
```

Comment:
How are you able to implement `Monad` with an `Applicative`. Would that not be a compile-time error?

Paging @mlutze Why is this not rejected? :)

Reply:
It's commented out. (Or am I missing something?)

Reply:
😝

Reply:
Welp, that was embarrassing.

Reply:
Okay, so this is resolved I guess? I just wanted to indicate that the instance implementation basically is there, but `Applicative` needs to be implemented first.